### DISCUSSION TITLE
split dns on linux roadwarrior with resovled ( respect attr 28675 ?)

### DISCUSSION THREAD

**Discussion by yannick** [strongswan/strongswan#2229] [Q&A] [answered]
i successfully made a manual config for a linux roadwarrior in much the same way what networkmanager gui generates.

I had to gerate my xfrm ipsec0 interface manually and add it to `/etc/strongswan.d/charon/resolve.conf`  (is there a way to dynamically create this interface according to the default if? ) 

however the last non working piece is that the (split) DNS domains are not forwarded to resolved. 
how do i do this?

my `charon/attr.conf` on the server:

```
attr {
    25 = cluster.local,svc.cluster.local
    2 = 172.20.0.10
    subnet = 172.20.0.0/16,10.44.0.0/16
    split-include = 172.20.0.0/16,10.44.0.0/16
    28674 = cluster.local
    28675 = cluster.local,svc.cluster.local
    28673 = 1
    28679 = 2
    load = yes
}
```

**Comment by tobiasbrunner** [comment, marked as answer]
> is there a way to dynamically create this interface according to the default if?

What do you mean?

> however the last non working piece is that the (split) DNS domains are not forwarded to resolved.
> how do i do this?

There is no support for this currently.

**Comment by yannick** [comment]
>is there a way to dynamically create this interface according to the default if?

by default no dedicated interface is/was created, it was putting the xfrm rules into the default? interface. 
what i naivly expected, that it would autocreate e.g. ipsec0 if i put that as the interface into the config if it didnt exist already. 
but i guess you'd add a script for that
